exec/jdbc-all/pom.xml

```diff
@@ -483,6 +518,269 @@
 
   <profiles>
     <profile>
+      <id>default</id>
```

Comment:
Not real happy about the amount of redundancy. But, I suppose Maven offers no alternative...